src/comm.jl

```diff
@@ -95,3 +95,10 @@ function Intercomm_merge(intercomm::Comm, flag::Bool)
     return newcomm
 end
 
+function Universe_size(comm::Comm)
+    rank = Ref{Cint}()
+    result = Ref(Ptr{Cvoid}(C_NULL))
+    @mpichk ccall((:MPI_Comm_get_attr, libmpi), Cint,
```

Comment:
I've started pasting the C signature as comments above the ccalls: it's been useful to ensure consistency.
```suggestion
    # int MPI_Comm_get_attr(MPI_Comm comm, int comm_keyval, void *attribute_val, int *flag)
    @mpichk ccall((:MPI_Comm_get_attr, libmpi), Cint,
```